modules/stochastic_tools/test/tests/reporters/statistics_reporter/gold/statistics_ci_out.json

```diff
@@ -0,0 +1,186 @@
+{
+    "time_steps": [
+        {
+            "reporters": [
```

Comment:
@zachmprince We should talk about the design of this output, there is a lot of duplication as is.